sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Text analytics SDK update the service to version `v3.0` from `v3.0-preview.1`.
 
 **Breaking changes**
+- Removed pagination feature, which removed `TextAnalyticsPagedIterable`, `TextAnalyticsPagedFlux` and `TextAnalyticsPagedResponse`
```

Comment:
oh wow! this is a big change. Do you mind sharing the reason behind it?

Reply:
As discussed over the phone call, with crews, The decision is to remove pagination feature.

Reply:
Talked offline